sdf/1.7/material.sdf

```diff
@@ -25,6 +25,10 @@
     </element>
   </element>
 
+  <element name="render_order" type="int" default="0" required="0">
+    <description>Set render order for coplanar planes</description>
```

Comment:
I think it's worth explaining how the order works. I assume the higher the value, the higher the priority, right?

Also, does this only apply for plane geometries or to any mesh triangle that is coplanar? Maybe that could be clarified too.